src/index.js

```diff
@@ -80,6 +81,17 @@ export const getCurrentNodeVersion = () => {
   return parseFloat(process.versions.node);
 };
 
+/**
+ * Returns the locally installed Electron version
+ * @return {string}
+ */
+export const getLocalElectronVersion = () => {
+  // Electron is not available in test environment so mock the version
+  if (process.env.NODE_ENV === "test") return "1.6.2";
```

Comment:
Can this just be mocked in the tests, I don't think it needs to be in the code. ^ I don't think you are using the method otherwise even though you made it an export?